docs/about/coding_guidelines.md

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
You also need to add this file to https://github.com/zio/zio/blob/master/website/sidebars.json